libs/expo/betterangels/src/lib/screens/NotesHmis/HmisLocation/LocationMapModal/Directions.tsx

```diff
@@ -0,0 +1,115 @@
+import { Colors, Spacings } from '@monorepo/expo/shared/static';
```

Comment:
line for line copy of `apps/betterangels/src/app/(private-screens)/add-note/Location/LocationMapModal/Directions.tsx`. Can we reuse a common component?